packages/app/src/cli/services/dev/output.ts

```diff
@@ -86,7 +86,7 @@ export function renderDev(renderConcurrentOptions: RenderConcurrentOptions, prev
       },
     }
   }
-  return renderConcurrent(options)
+  return renderConcurrent({...options, keepOpenAfterStopping: true})
```

Comment:
I get this change, but could we make it so extension only projects have a promise that never resolves intead?

Reply:
If not, maybe I'd change the name to something like `keepRunningAfterProcessesResolve`.